crates/xtask/src/flags.rs

```diff
@@ -52,6 +52,22 @@ xflags::xflags! {
             optional --verbose
         }
 
+        /// Starts a plex server to modify the test data.
+        cmd modify-data {
+            /// A tag from https://hub.docker.com/r/plexinc/pms-docker/tags to use for the server.
+            /// By default, `latest` is used.
```

Comment:
It seems like you've copied the flags from the `test` command, and because of that, the `--docker-tag` description is slightly incorrect. Currently, it says that the default is `latest`, but you're using the "minimal supported version".

Reply:
Ah yes, I had actually started with latest for simplicity but changed that later.